docs/data/base/components/select/UnstyledSelectBasic/css/index.js

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react';
+import Select, { selectClasses } from '@mui/base/Select';
+import Option, { optionClasses } from '@mui/base/Option';
+import { useTheme } from '@mui/system';
+
+export default function UnstyledSelectBasic() {
+  return (
+    <React.Fragment>
+      <Select
+        className="CustomSelect"
+        slotProps={{
+          listbox: { className: 'CustomListbox' },
+          popper: { className: 'CustomPopper' },
```

Comment:
```suggestion
          listbox: { className: 'CustomSelect-listbox' },
          popper: { className: 'CustomSelect-popper' },
```

Reply:
Or maybe we should use the generated class names after all. @michaldudak @siriwatknp what are your thoughts?